src/player.h

```diff
@@ -167,6 +181,7 @@ class Player
 	v3f m_speed;
 
 	std::vector<HudElement *> hud;
+	PlayerSettings m_player_settings;
```

Comment:
this should be private. You already have a getter to properly use it elsewhere. Please move that. m_* are intended to be private

Reply:
Done. The member is now private (not protected), uses in `LocalPlayer` have been replaced by a reference to the private member via getter.